src/nvim/statusline.c

```diff
@@ -1655,7 +1655,15 @@ int build_stl_str_hl(win_T *wp, char *out, size_t outlen, char *fmt, char *opt_n
           SignTextAttrs *sattr = virtnum ? NULL : &stcp->sattrs[i];
           p = sattr && sattr->text ? sattr->text : "  ";
           stl_items[curitem].minwid = -(sattr && sattr->text ? stcp->sign_cul_id ? stcp->sign_cul_id
-                                        : sattr->hl_id : (stcp->use_cul ? HLF_CLS : HLF_SC) + 1);
+                                                                                 : sattr->hl_id : (
```

Comment:
Will this look better if we put parens around the inner `(stcp->sign_cul_id ? stcp->sign_cul_id : sattr->hl_id)` as well and then reformat ? if not I would break that out to a variable just before.